features/sign_me_up_to_the_account.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Signing up to the account
+   as a User
+   In order to use the application
+   I would like to login
+
+   Background:
+      Given the following user exist
```

Comment:
If you use a data table, IMO you should write the step so that it can handle multiple users. In this case, I would use `the following users exist`.